python/psautohint/ufoFont.py

```diff
@@ -512,7 +513,7 @@ def readHashMap(self):
         if os.path.exists(hashPath):
             with open(hashPath, "rt") as fp:
                 data = fp.read()
-            newMap = eval(data)
+            newMap = ast.literal_eval(data)
```

Comment:
What is `data` and what is `newMap`?
(I ask because I'm wondering if it is really necessary to use even a `literal_eval` here.)

Reply:
`data` is a string and `newData` is a dictionary. (I alluded to it in the commit notes)